src/algebraic_geometry/prime_spectrum.lean

```diff
@@ -411,6 +403,13 @@ def basic_open (r : R) : topological_space.opens (prime_spectrum R) :=
 { val := { x | r ∉ x.as_ideal },
   property := ⟨{r}, set.ext $ λ x, set.singleton_subset_iff.trans $ not_not.symm⟩ }
 
+lemma std_aff_is_open (a : R) : is_open ({p : prime_spectrum R | a ∉ p.as_ideal }) :=
```

Comment:
Hmm, I see that the `def` above it is very similar... do you think you could use that instead of adding this lemma? It's a bundled open set.